statusboard/src/pages/Projects/utils.tsx

```diff
@@ -12,6 +12,30 @@ function projectGitHubCellLink(cell: Cell<Project>): JSX.Element {
   return <a href={project.code_url}>{project.name}</a>;
 }
 
+function projectWebsiteCellLink(cell: Cell<Project>): JSX.Element {
+  const project = cell.row.original;
+  if(project.link_url) {
+    return <a target='new' href={project.link_url}>website</a>;
+  }
+  return <span/>;
+}
+
+function projectOpenIssuesCell(cell: Cell<Project>): JSX.Element {
+  const project = cell.row.original;
+  if(project.open_issues_within) {
+    const issues = project.open_issues_within;
```

Comment:
I'd suggest you use something like 
```suggestion
  const openIssues = project.open_issues_within ?? '';
```

Reply:
Hum... or don't. The more I look at my own suggestion the less convinced I become 😛 
Whichever ends up being easier to implement/read I'd guess :/

Reply:
fixed with type number.